cmd/juju/controller/listmodels.go

```diff
@@ -227,16 +227,17 @@ type ModelSummary struct {
 	ShortName string `json:"short-name" yaml:"short-name"`
 	UUID      string `json:"model-uuid" yaml:"model-uuid"`
 
-	ControllerUUID     string              `json:"controller-uuid" yaml:"controller-uuid"`
-	ControllerName     string              `json:"controller-name" yaml:"controller-name"`
-	Owner              string              `json:"owner" yaml:"owner"`
-	Cloud              string              `json:"cloud" yaml:"cloud"`
-	CloudRegion        string              `json:"region,omitempty" yaml:"region,omitempty"`
-	ProviderType       string              `json:"type,omitempty" yaml:"type,omitempty"`
-	Life               string              `json:"life" yaml:"life"`
-	Status             *common.ModelStatus `json:"status,omitempty" yaml:"status,omitempty"`
-	UserAccess         string              `yaml:"access" json:"access"`
-	UserLastConnection string              `yaml:"last-connection" json:"last-connection"`
+	ControllerUUID     string                  `json:"controller-uuid" yaml:"controller-uuid"`
+	ControllerName     string                  `json:"controller-name" yaml:"controller-name"`
+	Owner              string                  `json:"owner" yaml:"owner"`
+	Cloud              string                  `json:"cloud" yaml:"cloud"`
+	CloudRegion        string                  `json:"region,omitempty" yaml:"region,omitempty"`
+	CloudCredential    *common.ModelCredential `json:"credential,omitempty" yaml:"credential,omitempty"`
```

Comment:
We seem to be reproducing "Cloud" in both the model information, and in the credential information. Is that useful? Is it just practical and fine? It doesn't seem to hurt anything, just odd to say the same thing 2x.

Reply:
Currently, our credentials are identified per user per cloud. We display cloud here as it is part of the credential information. I would not want to drop it here without dropping the cloud from the credential name.